File: sdkwork-rtc-sdk-csharp/providers/Sdkwork.Rtc.Sdk.Provider.Mediasoup/src/RtcProviderMediasoupPackageContract.cs
namespace Sdkwork.Rtc.Sdk.Provider.Mediasoup;

/// <summary>
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
/// </summary>
public static class RtcProviderMediasoupPackageContract
{
    public const string ProviderKey = "mediasoup";
    public const string PluginId = "rtc-mediasoup";
    public const string DriverId = "sdkwork-rtc-driver-mediasoup";
    public const string PackageIdentity = "Sdkwork.Rtc.Sdk.Provider.Mediasoup";
    public const string Status = "future-runtime-bridge-only";
    public const string RuntimeBridgeStatus = "reserved";
    public const bool RootPublic = false;
}

File: sdkwork-rtc-sdk-kotlin/providers/rtc-sdk-provider-mediasoup/src/main/kotlin/com/sdkwork/rtc/provider/mediasoup/RtcProviderMediasoupPackageContract.kt
package com.sdkwork.rtc.provider.mediasoup

/**
 * Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
 */
object RtcProviderMediasoupPackageContract {
    const val PROVIDER_KEY: String = "mediasoup"
    const val PLUGIN_ID: String = "rtc-mediasoup"
    const val DRIVER_ID: String = "sdkwork-rtc-driver-mediasoup"
    const val PACKAGE_IDENTITY: String = "com.sdkwork:rtc-sdk-provider-mediasoup"
    const val STATUS: String = "future-runtime-bridge-only"
    const val RUNTIME_BRIDGE_STATUS: String = "reserved"
    const val ROOT_PUBLIC: Boolean = false
}
